script/core/engines/cross-margin/cross-margin-engine/constructor-args.txt

```diff
@@ -0,0 +1 @@
+0x32076d5756edB4f1660946eeAA8d954e43141990 0x6694CB42C937C83677E01079a50468E33C2F7BfA
```

Comment:
can we not have this file stored locally? don't see why this would be useful in the future. Better just read from console input during verification

Reply:
This is useful for deploying new implementations that the proxy then gets upgraded to.

It could be read from input, but this is less prone to copy-paste errors